### PR TITLE
Database and coin type edit

### PR DESCRIPTION
**Pull request by nchengg**

### CODE REVIEW

File: src/gseproject/EXEplorer/models.py [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+class Tools(models.Model):
+    toolsID = models.AutoField(primary_key=True, verbose_name = "Tools ID")
+    toolsName = models.CharField(max_length=100, verbose_name = "Tools Name")
+    toolsDescription = models.TextField(verbose_name = "Tools Description")
+
+    def __str__(self):
+        return self.toolsName
+
+
+class Currency(models.Model):
```
Comment (Yout-bit) [inline src/gseproject/EXEplorer/models.py]: why is this a separte model to player? is it not a 1-1 realationship?
Could player just have fields:
user (str)
playerID (int)
cherries (int)

File: src/gseproject/EXEplorer/models.py [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
 from django.db import models
-from django.contrib.auth.models import User
 # Create your models here.
 
-class UserProfile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    coins = models.IntegerField(default=0)
\ No newline at end of file
+class Player(models.Model):
+    user = models.CharField(settings.AUTH_USER_MODEL, on_delete = models.CASCADE, verbose_name = "User", related_name = "players")
```
Comment (Yout-bit) [inline src/gseproject/EXEplorer/models.py]: Is this ther username field, as it's only a string? If so can it be renammed to username

File: src/gseproject/EXEplorer/models.py [hunk 3]
```diff
@@ -1,7 +1,41 @@
...
+    cherry = models.PositiveIntegerField(default = 0, verbose_name = "cherry")
+    
+    def __str__(self):
+        return f"{self.cherry} cherries - {self.player}"
+
+
+
+class Tree(models.Model):
+    player = models.ForeignKey(Player, on_delete = models.CASCADE, verbose_name = "Player", related_name = "trees")
+    tree = models.PositiveIntegerField(default = 0, verbose_name = "tree")
```
Comment (Yout-bit) [inline src/gseproject/EXEplorer/models.py:24]: What does the field tree represent?

**Review by Yout-bit — CHANGES_REQUESTED** [review]

File: src/gseproject/EXEplorer/models.py (hunk 3 above)
Comment (nchengg) [reply src/gseproject/EXEplorer/models.py:24]: We plant trees in game. Tree represents 4 states (seed, sprout ......)

File: src/gseproject/EXEplorer/models.py (hunk 2 above)
Comment (nchengg) [reply src/gseproject/EXEplorer/models.py]: edited

**Pull request closed without merging**